### PR TITLE
Update SRM_migrating_profile_instances.md

### PR DESCRIPTION
**Pull request by philipredbee**

### CODE REVIEW

File: user-guide/Standard_Apps/SRM/SRM_migrating/SRM_migrating_profile_instances.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ uid: SRM_migrating_profile_instances
 To export and import profile instances, you can use the *PFM_ProfileInstancesImportExport* Automation script.
 
 > [!IMPORTANT]
-> The *PFM_ProfileInstancesImportExport* script needs to be able to generate an .xlsx file. For this reason, before you use the script, you need to make sure *AccessDatabaseEngine.exe* is installed in the DMS. You can download this from [microsoft.com](https://www.microsoft.com/en-in/download/details.aspx?id=54920).
+> The *PFM_ProfileInstancesImportExport* script needs to be able to generate an .xlsx file. For this reason, before you use the script, you need to make sure the x86 version of *AccessDatabaseEngine.exe* is installed in the DMS. You can download this from [microsoft.com](https://www.microsoft.com/en-in/download/details.aspx?id=54920).
```
Comment (slc-peter-vanpoucke) [inline user-guide/Standard_Apps/SRM/SRM_migrating/SRM_migrating_profile_instances.md:10]: @EdithLansens could you rephrase? So to be more in line with what's on the download page.
> the 32-bit (x86) version of

**Review by slc-peter-vanpoucke — CHANGES_REQUESTED** [review]
Correct. Added a small suggestion.

File: user-guide/Standard_Apps/SRM/SRM_migrating/SRM_migrating_profile_instances.md (hunk 1 above)
Comment (philipredbee) [reply user-guide/Standard_Apps/SRM/SRM_migrating/SRM_migrating_profile_instances.md]: I've updated with a new commit

**Review by EdithLansens — APPROVED** [review]
